frontend/test/__support__/e2e/helpers/e2e-email-helpers.js

```diff
@@ -53,19 +53,21 @@ export const clearInbox = () => {
 };
 
 export const openEmailPage = emailSubject => {
-  cy.window().then(win => (win.location.href = "http://localhost"));
+  const webmailInterface = `http://localhost:${WEB_PORT}`;
+
+  cy.window().then(win => (win.location.href = webmailInterface));
   cy.findByText(emailSubject).click();
 
   return cy.hash().then(path => {
-    const htmlPath = `http://localhost${path.slice(1)}/html`;
+    const htmlPath = `${webmailInterface}${path.slice(1)}/html`;
     cy.window().then(win => (win.location.href = htmlPath));
     cy.findByText(emailSubject);
   });
 };
 
 export const clickSend = () => {
   cy.button("Send email now").click();
-  cy.button("Email sent", 30000);
+  cy.button("Email sent", 60000);
```

Comment:
Does email start to take this much time? Is this changing from 30s timeout to 60s? Does this also implies our email tests run slower these days?

Reply:
To be determined